src/Components/Publishing/ToolTip/ToolTip.tsx

```diff
@@ -2,10 +2,11 @@ import styled from "styled-components"
 import React from "react"
 import { ArtistTooltipContainer } from "./ArtistToolTip"
 import { GeneToolTipContainer } from "./GeneToolTip"
-import { ArrowDown, ArrowContainer } from "./Components/ArrowDown"
+import { Arrow, ArrowContainer } from "./Components/Arrow"
 
 interface Props extends React.HTMLProps<HTMLDivElement> {
   entity: object
+  isBelowContent?: boolean
```

Comment:
just a suggestion but I would use `position: "up" | "down"` as the API for placing the tooltip instead of a boolean here as I think it's clearer.